### ISSUE TITLE
RAdam ignores weight decay

### ISSUE THREAD

**Issue by black-and-blueish** [fastai/fastai2#246]
I noticed that the `wd` parameter in `Learner `does not have any effect on training when using `RAdam` optimizer. Looking at the code it seems obvious why:
```
    cbs = [weight_decay] if decouple_wd else [l2_reg]
    cbs = [partial(average_grad, dampening=True), average_sqr_grad, step_stat, radam_step]

```
Should not the last line be (same as in `Adam`):
`cbs += [partial(average_grad, dampening=True), average_sqr_grad, step_stat, radam_step]`

**Comment by sgugger** [comment]
Yes, it's a mistake. Would you mind making a PR to fix this?
